src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -117,24 +118,23 @@ private void commaSeparatedEntryTransformer(String commaSeparatedEntryKey,
         exceptionRecordToCcdFieldsMap.put(OcrFieldName.CHILD_SUPPORT_AGENCY_CALCULATION_MADE, "ChildSupportAgencyCalculationMade");
         exceptionRecordToCcdFieldsMap.put(OcrFieldName.CHILD_SUPPORT_AGENCY_CALCULATION_REASON, "ChildSupportAgencyCalculationReason");
 
-        exceptionRecordToCcdFieldsMap.put("ApplicantRepresented", "applicantRepresentPaper");
-        exceptionRecordToCcdFieldsMap.put("ApplicantSolicitorName", "solicitorName");
-        exceptionRecordToCcdFieldsMap.put("ApplicantSolicitorFirm", "solicitorFirm");
-        exceptionRecordToCcdFieldsMap.put("ApplicantSolicitorPhone", "solicitorPhone");
-        exceptionRecordToCcdFieldsMap.put("ApplicantSolicitorDXnumber", "solicitorDXnumber");
-        exceptionRecordToCcdFieldsMap.put("ApplicantSolicitorReference", "solicitorReference");
-        exceptionRecordToCcdFieldsMap.put("ApplicantPBANumber", "PBANumber");
-        exceptionRecordToCcdFieldsMap.put("ApplicantSolicitorEmail", "solicitorEmail");
-        exceptionRecordToCcdFieldsMap.put("ApplicantPhone", "applicantPhone");
-        exceptionRecordToCcdFieldsMap.put("ApplicantEmail", "applicantEmail");
-
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_REPRESENTED, "applicantRepresentPaper");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_SOLICITOR_NAME, "solicitorName");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_SOLICITOR_FIRM, "solicitorFirm");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_SOLICITOR_PHONE, "solicitorPhone");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_SOLICITOR_DX_NUMBER, "solicitorDXnumber");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_SOLICITOR_REFERENCE, "solicitorReference");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_PBA_NUMBER, "PBANumber");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_SOLICITOR_EMAIL, "solicitorEmail");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_PHONE, "applicantPhone");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_EMAIL, "applicantEmail");
         return exceptionRecordToCcdFieldsMap;
     }
 
-    private void applyMappingsForAddress(String prefix, String unusualField, List<OcrDataField> ocrDataFields, Map<String, Object> modifiedMap) {
+    private void applyMappingsForAddress(String prefix, String parentField, List<OcrDataField> ocrDataFields, Map<String, Object> modifiedMap) {
         String nestedFieldPrefix = StringUtils.capitalize(prefix + "Address");
         addMappingsTo(
-            unusualField,
+            parentField,
```

Comment:
:+1:

Reply:
who has used such a useless name!?

Reply:
they were moved to `shouldPassValidationForValidMandatoryAndOptionalFields`